### PR TITLE
Update CHANGELOG for 1.3.1 release

### PR DESCRIPTION
**Pull request by amartinezfayo**
Update CHANGELOG for 1.3.1 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+- The `windows` workload attestor gained a new `sha256` selector that can attest the SHA256 digest of the workload binary (#3100)
+
+### Fixed
+- All records related with registration entries are now also deleted when a registration entry is deleted (#3127)
+- Removed orphaned database records related with registration entries (#3132)
+- Re-introduced entry output mask on agent FetchX509SVID sync with the server (#3123)
+- Issue with read-modify-write operations when using PostgreSQL datastore in hot standby mode (#3103)
+
+### Changed
+- FetchX509Bundles RPC now only sends the updates when there are new bundles, preventing spurious updates (#3102)
```
Comment (azdagron) [inline CHANGELOG.md:15]:
```suggestion
- FetchX509Bundles RPC no longer sends spurious updates that contain no changes (#3102)
```

**Review by azdagron — COMMENTED** [review]
Thanks for pulling this together, @amartinezfayo !

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
 
+## [1.3.1] - 2022-06-09
+
+### Added
+- The `windows` workload attestor gained a new `sha256` selector that can attest the SHA256 digest of the workload binary (#3100)
+
+### Fixed
+- All records related with registration entries are now also deleted when a registration entry is deleted (#3127)
+- Removed orphaned database records related with registration entries (#3132)
+- Re-introduced entry output mask on agent FetchX509SVID sync with the server (#3123)
```
[the remark this answers is not in the document]
Comment (amartinezfayo) [reply CHANGELOG.md:11]: Good catch!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+- Issue with read-modify-write operations when using PostgreSQL datastore in hot standby mode (#3103)
+
+### Changed
+- FetchX509Bundles RPC now only sends the updates when there are new bundles, preventing spurious updates (#3102)
+- Warn if the built-in `join_token` node attestor is attempted to be overridden by an external plugin (#3045)
+- Datastore is now closed when the catalog is shut down and logging was improved (#3047)
+- Updated the CRD registrar quickstart to use SPIRE 1.3.0, and small documentation improvements (#3061)
```
[the remark this answers is not in the document]
Comment (amartinezfayo) [reply CHANGELOG.md:18]: I was hesitant also. I'll remove it :)

**Review by azdagron — APPROVED** [review]

**Review by MarcosDY — APPROVED** [review]
LGTM!!!

**Pull request merged**